swap_meet/item.py

```diff
@@ -1,2 +1,16 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.id = uuid.uuid1().int if not id else id
+        self.condition = condition
+
+    def get_category(self):
+        return self.__class__.__name__
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}"
+
+    def condition_description(self):
+        descriptions = {0: "torn", 1: "a bit shredded", 2: "defs used", 3: "lightly used", 4: "near perfect", 5: "perf",}
```

Comment:
nice use of a `dict` to store this info, vs an if statement.  You can imagine that an if statement would take worst case O(N) to fall through to the last condition. ;)